YTDemo/ViewController.swift

```diff
@@ -0,0 +1,696 @@
+//
+//  ViewController.swift
+//  YTDemo
+//
+//  Created by Rukesh Prajapati on 7/11/17.
+//  Copyright © 2017 callistos. All rights reserved.
+//
+
+import UIKit
+
+/*class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UITextFieldDelegate {
+ 
+ var videosArray: [[String: AnyObject]] = []
+ 
+ var apiKey = "AIzaSyBLsZ4ZurTYWZMGDuEtn6C-6OmqgU5emE0"
+ 
+ var desiredChannelsArray = ["Apple", "Google", "Microsoft"]
+ 
+ var channelIndex = 0
+ 
+ var channelsDataArray: [[String: AnyObject]] = []
+ 
+ var selectedVideoIndex: Int!
+ 
+ @IBOutlet weak var tblVideos: UITableView!
+ 
+ 
+ @IBOutlet weak var viewWait: UIActivityIndicatorView!
+ 
+ @IBOutlet weak var segDisplayedContent: UISegmentedControl!
+ 
+ @IBOutlet weak var searchTextField: UITextField!
+ 
+ /// <#Description#>
+ override func viewDidLoad() {
+ super.viewDidLoad()
+ 
+ tblVideos.delegate = self
+ tblVideos.dataSource = self
+ searchTextField.delegate = self
+ 
+ //        let nib = UINib(nibName: "VideosTableViewCell", bundle: nil)
+ //        tblVideos.register(nib, forCellReuseIdentifier: "VideosTableViewCell")
+ let nib = UINib(nibName: "VideosTableViewCell", bundle: nil)
+ tblVideos.register(nib, forCellReuseIdentifier: "VideosTableViewCell")
+ let nib2 = UINib(nibName: "ChannelTableViewCell", bundle: nil)
+ tblVideos.register(nib2, forCellReuseIdentifier: "ChannelTableViewCell")
+ 
+ getChannelDetails(useChannelIDParam: false)
+ 
+ }
+ 
+ override func didReceiveMemoryWarning() {
+ super.didReceiveMemoryWarning()
+ // Dispose of any resources that can be recreated.
+ }
+ 
+ func numberOfSectionsInTableView(tableView: UITableView) -> Int {
```

Comment:
this function is not required if there is only one section in table view , so remove it @prajapatirukesh3c